main.go

```diff
@@ -301,6 +334,14 @@ func main() {
 		}
 		kubectlPortForward(pod.Name, localPort, remotePort, namespace)
 	}
+	if proxyType == ProxyTypeSQL {
+		sqlInstance := readCloudSQLInstance(projectID)
+		localPort := readLocalPort(3306)
```

Comment:
I would use the same default port as the Cloud SQL instance if possible.

Reply:
ive closed this PR in favor of https://github.com/AckeeCZ/goproxie/pull/29 before you took those comments, but we can finish those discussions here i gues, it havent changed much, but i havent found any way to convert existing PR to WIP 😂 

To the point, according to [original source here](https://github.com/GoogleCloudPlatform/cloudsql-proxy/blob/master/cmd/cloud_sql_proxy/cloud_sql_proxy.go#L530) which leads to [here](https://github.com/GoogleCloudPlatform/cloudsql-proxy/blob/master/proxy/proxy/client.go#L64), that would be always 3307 the way I see it. Or what do I miss? 🤔

Reply:
I am not sure what do they mean by that comment, but 3306 is MySQL default port and it does not make sense to use it if the database is not MySQL. I think using 3306 for Postgres, for example, would be the same as using port 80 for SSH. It would work, but it is ugly 🙂

Reply:
Will look into it. In the meantime as a workaround, how would you like a random of the two (as it can only by mysql and pg) :D Random seed will be based on the name to be consistent in time, and it could be right in 50%  cases 😌

Reply:
> as it can only by mysql and pg

I think Cloud SQL supports Microsoft SQL server as well 🙂

Reply:
Right. 👀

Reply:
Added and ready for review here: https://github.com/AckeeCZ/goproxie/pull/29